app/Platform/Services/GameTopAchieversService.php

```diff
@@ -134,7 +134,7 @@ public static function expireTopAchieversComponentData(int $gameId): void
 
     public function getTopAchieversComponentData(): array
     {
-        $cacheKey = "game:{$this->gameId}:top-achievers";
+        $cacheKey = "game:{$this->gameId}:top-achievers:v2-1";
```

Comment:
Additional changes to this key should only affect you and I. Recommend keeping it as `:v2`.

Reply:
Reverted back to `:v2` in latest.